app/services/notifications/new_comment/send.rb

```diff
@@ -40,8 +40,9 @@ def call
 
         PushNotifications::Send.call(
           user_ids: targets,
-          title: "@#{comment.user.username}",
-          body: "Re: #{comment.parent_or_root_article.title.strip}",
+          title: "💬 New Comment",
+          body: "#{comment.user.username} commented on " \
+                "#{comment.commentable.title.strip}:\n#{comment.body_markdown.strip}",
```

Comment:
Are notifications on the iOS side ok with Markdown? Or would it maybe be better to use something like `helper.strip_tags(comment.processed_html)`?

Reply:
TIL about the [strip_tags](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-strip_tags) helper and I agree we should likely use it on `processed_html` to get rid of tags because markdown won't look too good on PN payloads.

This is what [this comment](https://dev.to/admin/blazer/queries/534-markdown-heavy-comment) looks like after trying to "clean them":

<img width="687" alt="Screen Shot 2021-09-08 at 08 57 55" src="https://user-images.githubusercontent.com/6045239/132533755-a7bcf2fd-ca5f-41ee-a1f0-f5417f78f143.png">

(`var` has the `body_markdown` and `var2` has `processed_html`)

Reply:
Wonderful, thanks much for the suggestion @citizen428! I added `strip_tags`, and comments with markdown look much better!